.github/workflows/build_publish_image_release.yml

```
@@ -0,0 +1,58 @@
+name: 📦 Build and Publish a WBS Image Release to Dockerhub
+
+on:
+  push:
+    tags:
+      - 'elasticsearch@*'
+      - 'quickstatements@*'
+      - 'wdqs@*'
+      - 'wdqs-frontend@*'
+      - 'wdqs-proxy@*'
+      - 'wikibase@*'     
+
+permissions:
+  contents: write
+  packages: write
+
+jobs:
+  extract_and_use_tag:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Extract imageName and confirm it is a valid semantic version tag
+        id: extract_tag
+        run: |
+          # Extract the tag name
+          TAG_NAME=$(echo ${{ github.ref }} | sed 's/refs\/tags\///')
+
+          # Use regex to extract the imageName before the @ and confirm the version is valid semver version tag
+          if [[ "$TAG_NAME" =~ ^([^@]+)@([0-9]+\.[0-9]+\.[0-9]+(-[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?(\+[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?)$ ]]; then
+            # Sets and passes the imageName output from this step, which is used in the subsequent step
+            # e.g. `steps.extract_tag.outputs.imageName` gets set to "wikibase" matched from "wikibase@1.01")
```

```suggestion
            # e.g. `steps.extract_tag.outputs.imageName` gets set to "wikibase" matched from "wikibase@1.0.1")
```
I think this is a typo.